pytensor/link/pytorch/dispatch/scalar.py

```diff
@@ -19,9 +22,14 @@ def pytorch_funcify_ScalarOp(op, node, **kwargs):
     if nfunc_spec is None:
         raise NotImplementedError(f"Dispatch not implemented for Scalar Op {op}")
 
-    func_name = nfunc_spec[0]
+    func_name = nfunc_spec[0].replace("scipy.", "")
```

Comment:
Thinking about #1032 this will now not work for scipy function. Perhaps inside the Elemwise dispatch we should have this logic. And if it doesn't work only then do we funcify the ScalarOp + nested vmap?

Reply:
So I guess this technically _works_ if you just run the test, but I think what you're saying is that we don't do what we expect, e.g. call the scipy methods torch has with all data since it supports it's own broadcasting mechanism. Fixing it now.